data/mods/Magiclysm/Spells/item_only.json

```
@@ -103,5 +103,38 @@
     "spell_class": "NONE",
     "message": "Your veins feel like they are on fire!\nYou start regenerating mana!",
     "extra_effects": [ { "id": "potion_recover_mana" } ]
+  },
+  {
+    "id": "lupercalia",
+    "type": "SPELL",
+    "name": "Lupercalian Release",
+    "//": "Used for wolfshead cufflinks not castable",
+    "description": "This spell transforms you into a werewolf. There may be some cons to this.",
+    "valid_targets": [ "self" ],
+    "flags": [ "SILENT", "NO_LEGS", "NO_HANDS" ],
+    "effect": "spawn_item",
+    "effect_str": "aura_werewolf",
+    "max_level": 1,
+    "min_damage": 10,
+    "max_damage": 10,
+    "damage_increment": 0,
+    "damage_type": "bio",
+    "difficulty": 0,
+    "spell_class": "NONE",
+    "effect": "target_attack",
+    "base_casting_time": 300,
+    "base_energy_cost": 2,
+    "min_duration": 43200,
+    "max_duration": 43200,
+    "extra_effects": [ { "id": "werewolf_healing" }, { "id": "pain_split" }, { "id": "dark_sight" } ]
```

```suggestion
    "extra_effects": [ { "id": "pain_split" }, { "id": "werewolf_healing" } ]
```
why cast dark sight? just give the werewolf enchantment night vision.
Also, ectra_effects are cast in order, so you'd want to pain split before healing.